nautobot/dcim/views.py

```diff
@@ -3143,3 +3144,130 @@ class PowerFeedBulkDeleteView(generic.BulkDeleteView):
     queryset = PowerFeed.objects.prefetch_related("power_panel", "rack")
     filterset = filters.PowerFeedFilterSet
     table = tables.PowerFeedTable
+
+
+class ReliabilityGroupEditView(generic.ObjectEditView):
```

Comment:
Since this is net new, can you please try to use the `NautobotViewSet` machinery instead of the legacy generic views?

Reply:
Needed to overload, default methods for the inline formset factory. Happy to move to viewset just need to know where it goes.